netlify.toml

```diff
@@ -3,7 +3,7 @@ publish = "public"
 command = "make production-build"
 
 [build.environment]
-HUGO_VERSION = "0.55.0"
+HUGO_VERSION = "0.62.0"
```

Comment:
It also needs to be updated within the Makefile. It might be worth doing something similar to the [kubernetes/website Makefile](https://github.com/kubernetes/website/blob/master/Makefile#L2) and have `netlify.toml` be the source of truth. That way we don't need to update it in two places.